src/home/search.vue

```diff
@@ -17,11 +17,10 @@
       input(placeholder="Search text")
 
       menu#search1.min
-        header.fixed
-          .field.large.prefix.suffix.no-margin
-            i.front(data-ui="#search1") arrow_back
-            input(placeholder="Search text")
-            i.front(data-ui="#search1") close
+        .field.large.prefix.suffix.no-margin.fixed
```

Comment:
Im facing some trouble here using this search. when i click and instantly type something the i cant delete the text typed from the first input

![image](https://github.com/beercss/beercss/assets/43939417/08989165-0075-479e-83fe-b39a184016a2)

Reply:
As a CSS solution, I think we can't change field values with JS. Some frameworks can't update the value correctly if we change that. With modern JS frameworks, I suggest to use the same property bind on both fields. Ok?

Reply:
I got the idea, im just concerned with the user experience on beercss site.
Is it possible to have only one input ? If we have the same idea with just one input it would be perfect.

Reply:
Agree with it. How about to create another feature PR to try change this behaviour? I have no ideas at this moment to do that.